features/intl-duration-format.yml

```
@@ -0,0 +1,11 @@
+name: Intl.DurationFormat
+description: "The `Intl.DurationFormat` API is a locale-aware formatter that can format a duration object according to locales and styles."
```

A suggestion for consistency with `Intl.Segmenter` and https://github.com/web-platform-dx/web-features/pull/1580#discussion_r1715400547, and to avoid a "circular" description (i.e., format is a formatter that formats 😄):

```suggestion
description: "The `Intl.DurationFormat` API creates a locale-aware formatter that turns an object representing a duration (such as days, hours, and minutes) into a string."
```